src/observers/OmnisharpServerStatusObserver.ts

```diff
@@ -0,0 +1,55 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
How about we call this the `ShowWarningMessageObserver`?

Reply:
Done!